### DISCUSSION TITLE
Some subtle improvements with a big impact

### DISCUSSION THREAD

**Discussion by Erevoid** [Orama-Interactive/Pixelorama#402] [Ideas]
Here are some things that I think would improve the user experience and may not require much work:

- [x] Being able to paste something without having an active selection (either by pasting it on the top left of the canvas or in the middle of where the user is looking at)
- [x] When creating a new image, let's say 64x128, there should be two icons indicating portrait and landscape. If your image is a portrait and you press the landscape icon the height and width are switched. 
- [x] While cloning a frame, check whether the link cels is on and if it is, maintain it for the cloned cels.
- [x] Just like the brushes, textures for the bucket fill should exist when copying something, for a fast fill with a texture you just made (project brushes-project textures)
- [ ] When opening the software and then opening another project (not from file>open) made with pixelorama, it opens two seperate windows instead of an extra tab. This makes it uncomfortable to copy and paste something between projects, if they are in a different window

**Comment by OverloadedOrama** [comment]
> While cloning a frame, check whether the link cels is on and if it is, maintain it for the cloned cels.

I don't know if I agree with this. Adding a new frame when the linked cel button is enabled already adds the new cel as linked. If we add this suggestion, then there would be no difference between cloning and adding, when the linked cel button is enabled. I think I'm more in favor of keeping the current behavior because it allows the user to quickly clone a frame without making it remain linked.

**Reply by OverloadedOrama** [reply]
Nevermind, I gave it more thought and I realized that that suggestion would be indeed useful for when the user has multiple layers and only wants some of them to have linked cels. Suggestion is now implemented in c6a056bba664fed4c15bd1a388a819d0d0bab91a.
